### ISSUE TITLE
Sidebar is not closing properly

### ISSUE THREAD

**Issue by SantoshPattnaik** [microsoft/vscode#136294]
Does this issue occur when all extensions are disabled?: Yes/No
No

- VS Code Version: Version: 1.61.2 stable build(user setup)
Commit: 6cba118ac49a1b88332f312a8f67186f7f3c1643

- OS Version: Windows NT x64 10.0.22000 (Windows 11 Beta build) 

Steps to Reproduce:

1.Open the sidebar and close it after some work

![Vs code bug](https://user-images.githubusercontent.com/58502105/139866961-8809aba8-5373-466d-91e4-8d5aa74f44ba.jpg)

**Comment by sbatten** [comment]
@deepak1556 have you seen anything like this reported. The sidebar seems properly hidden here since I can see the line numbers of the editor. Some gpu acceleration screen cheese perhaps?

**Comment by SantoshPattnaik** [comment]
can anybody tell what more info can I give?

**Comment by deepak1556** [comment]
Does the issue repro when you start with `code --disable-gpu` ?

**Comment by SantoshPattnaik** [comment]
Actually this problem happen very often and I noticed this when i closed the sidebar. This problem doesn't occur always. As it seemed odd to me I informed. 

Whether I use `--disable-gpu` or just start it normally the problem not happening now. 

So what should I do now?

**Comment by SantoshPattnaik** [comment]
https://user-images.githubusercontent.com/58502105/141681424-918b272c-ab14-404f-a6be-84a6c653f9e8.mp4

**Comment by SantoshPattnaik** [comment]
Guys I found that this problem enables when I use github dark default theme and reload the window with the panel opened. When the reload completes properly I close the panel and the issue recreates.

See This:

https://user-images.githubusercontent.com/58502105/141682558-2ef14d95-93d2-4bbe-bcac-bdf945942f79.mp4

**See this in fullscreen to have a better look at the problem**

**Comment by SantoshPattnaik** [comment]
> Hey @deepak1556 sbatten, this issue might need further attention.
> 
> @SantoshPattnaik, you can help us out by closing this issue if the problem no longer exists, or adding more information.

@vscode-triage-bot @deepak1556 @sbatten 

Sir the problem now also exists and the problem can be reproduced.

This prblm happens when I open the sidebar and click on any other button on the sidebar like extensions, search or anything which changes the sidebar width, after clicking on the button through which I first opened the sidebar, and closing it the prblm is reproduced.
You can refer the video (Vscode1.mp4)

**Comment by sbatten** [comment]
@SantoshPattnaik now that you have a repro again, can you test with `--disable-gpu`

**Comment by SantoshPattnaik** [comment]
> @SantoshPattnaik now that you have a repro again, can you test with `--disable-gpu`

YES, the prblm is gone after i uncommented disable-gpu-acceleration flag in argv.json.

I have opened and closed vs code 10 times after adding the flag and the prblm has gone.

After I restored the settings, the problm is happening 6-7 times in the 10 times open and close period.

I hope you understand
Well may I know is this a software problem or operating system issue as other software windows are rendering fine?
